### PR TITLE
Add 대성마이맥(mimacstudy) Presence

### PR DESCRIPTION
**Pull request by 1LUXION**
> **대성마이맥(mimacstudy)** is the Internet lecture service in South Korea.
> I made the presence of 대성마이맥(mimacstudy) in order to motivate students of Korea,
> so it may be difficult to review my code since it contains several strings written in Korean.
> I'm sorry for the inconvenience, and I appreciate reviewers' hard work.

### Home Screen
home

###  Lecture Room
lecture

###  Teacher Room
teacher

###  Lecture Paused
lecture paused

###  Lecture Playing
lecture playing

### CODE REVIEW

File: websites/#/대성마이맥/presence.ts [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+
+    else if (document.location.pathname.includes("/guhae/guhaeMain.ds"))
+      presenceData.details = "구해";
+
+    else 
+      presenceData.details = "진짜 공부는 지금부터";
+    
+  }
+
+  if (presenceData.details === null) {
```
Comment (SlowLife1661) [inline websites/#/대성마이맥/presence.ts:134]:
```suggestion
  if (!presenceData.details) {
```

**Review by Slowlife01 — APPROVED** [review]

File: websites/#/대성마이맥/presence.ts (hunk 1 above)
Comment (SlowLife1661) [inline websites/#/대성마이맥/presence.ts:134]:
```suggestion
  if (!presenceData.details) {
```

**Review by Slowlife01 — CHANGES_REQUESTED** [review]

**Comment by 1LUXION** [comment]
@SlowLife1661 Committed Requested Change

**Review by Slowlife01 — APPROVED** [review]

**Review by Bas950 — APPROVED** [review]
LGTM

**Pull request merged**